tests/test_events.py

```diff
@@ -338,6 +338,57 @@ def test_event_is_upcoming(event, mocker):
         assert not test_event.is_upcoming
 
 
+@freeze_time("2021-02-01 10:00:00")
+def test_first_of_month_returns_no_recent_past_meetings(event):
+    yesterday = LAMetroEvent._time_ago(days=1).strftime('%Y-%m-%d %H:%M')
+    yesterdays_event = event.build(
+        name='Board Meeting', start_date=yesterday, id=get_event_id())
+
+    assert LAMetroEvent.most_recent_past_meetings().count() == 0
+
+
+@freeze_time("2021-02-07 10:00:00")
+def test_most_recent_past_meetings(event):
+    three_weeks_ago = LAMetroEvent._time_ago(
+        days=21).strftime('%Y-%m-%d %H:%M')
+    one_week_from_now = LAMetroEvent._time_from_now(
+        days=7).strftime('%Y-%m-%d %H:%M')
+    one_minute_from_now = LAMetroEvent._time_from_now(
+        minutes=1).strftime('%Y-%m-%d %H:%M')
+    eight_days_ago_last_month = LAMetroEvent._time_ago(
+        days=8).strftime('%Y-%m-%d %H:%M')
+
+    four_days_ago = LAMetroEvent._time_ago(days=4).strftime('%Y-%m-%d %H:%M')
+    five_days_ago = LAMetroEvent._time_ago(days=5).strftime('%Y-%m-%d %H:%M')
```

Comment:
Maybe this is an insane request, 😝, but can you put these in chronological order?

Reply:
Of course!